src/main/java/chatbot/commands/EventCommand.java

```diff
@@ -0,0 +1,61 @@
+package chatbot.commands;
+
+import chatbot.storage.Storage;
+import chatbot.tasks.Event;
+import chatbot.tasks.Task;
+import chatbot.tasks.TaskList;
+import chatbot.ui.Ui;
+
+/**
+ * Represents a command to add an event task to the task list
+ */
+public class EventCommand extends Command {
+    /**
+     * Description of the event task to be added
+     */
+    String description;
+
+    /**
+     * Start date of the event
+     */
+    String from;
+
+    /**
+     * End date of the event
+     */
+    String to;
```

Comment:
This might need access modifiers to be more explicit
```suggestion
    /**
     * Description of the event task to be added
     */
    private String description;

    /**
     * Start date of the event
     */
    private String from;

    /**
     * End date of the event
     */
   private  String to;
```